Text/PresidioPIIAnonymization/README.md

```diff
@@ -0,0 +1,218 @@
+    variable "docker_registry_password" {
+      description = "[your container registry password]"
+      default = ""
+    }
+    
+    variable "docker_image" {
+      description = "[your docker image name]:[your tag]"
+      default = ""
+    }
+    
+    variable "resource_group" {
+      description = "This is the name of an existing resource group to deploy to"
+      default = ""
+    }
+    
+    variable "location" {
+      description = "This is the region of an existing resource group you want to deploy to"
+      default = "eastus2"
+    }
+    
+    variable "debug" {
+      description = "API logging - set to True for verbose logging"
+      default = false
+    }
+
+    ```
+    
+    Navigate to the directory containing the files and enter:
+    
+    ```bash
+    terraform init
+    ```
+    Then enter:
+    ```bash
+    terraform apply
+    ```
+    You will be prompted with:
+    
+    ```bash
+    Do you want to perform these actions?
+      Terraform will perform the actions described above.
+      Only 'yes' will be accepted to approve.
+    ```
+    
+    Type ```bash yes```
+    
+    Once deployed, copy the Azure Web App URL which may be found in the overview section of the portal as we will need 
+    it to plug into Azure Search.
+    
+1) ### Deploy the datasource, index, skillset and indexer
+
+   #### Data source
+   
+    Populate your values in the [data source file](deployment/azuresearch/create_data_source.json) or use the 
+    [Create the data source](notebooks/PII%20Anonymization.ipynb#Create-the-data-source)
+
+    #### Index
+    Populate your values in the [index file](deployment/azuresearch/create_index.json) or use the 
+    [Create the index](notebooks/PII%20Anonymization.ipynb#Now-we-create-the-index)
+    
+    #### Skillset
+    
+    Populate the values in the [skillset file](deployment/azuresearch/create_skillset.json) or use the 
+    [Create the SkillSet](notebooks/PII%20Anonymization.ipynb#Now-we-create-the-skill-set)
+      
+    Note, you need an already deployed ACS instance in the same region as your cognitive services
+    instance as we want to augment what we can extract using custom vision with our similarity
+    model.
+    
+    You will need your [ACS API Key](https://docs.microsoft.com/en-us/azure/search/search-security-api-keys)
+    and the URL for your ACS instance. 
+   
+1) ### Run the ACS indexer 
+
+    Populate the values in the [indexer file](deployment/azuresearch/create_indexer.json) or 
+    [Create/Run your indexer](notebooks/PII%20Anonymization.ipynb#Now-we-create-the-indexer)
+
+    The indexer will automatically run and you should see requests coming in if you look at the Web App logs.
+
+1)  ### Test the index 
+    Investigate your indexed data, check the most similar images
+
+    Now we are in a position to search on our most similar data, navigate to the [Let's go and test the ACS index](notebooks/PII%20Anonymization.ipynb#Let's-go-and-test-the-ACS-index)
```

Comment:
Add quotes and specify the type of thing we're navigating to.

Reply:
done.